foodsaving/locale/ar/LC_MESSAGES/django.po

```diff
@@ -298,7 +298,7 @@ msgstr ""
 
 #: foodsaving/groups/templates/group_summary.html.jinja2:171
 #, python-format
-msgid "%(weight)s kg from <a href=\"%(store_url)s\">%(store_name)s</a>"
+msgid "%(weight)s kg from <a href=\"%(place_url)s\">%(place_name)s</a>"
```

Comment:
Changes of msgid will probably invalidate translations of affected messages. We should tell translators before we release.